client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -16,28 +16,28 @@ import QueryPlans from 'components/data/query-plans';
 
 const CALYPSO_JETPACK_CONNECT = '/jetpack/connect';
 
-const PlansLanding = React.createClass( {
+class PlansLanding extends Component {
 
-	propTypes: {
-		sites: React.PropTypes.object,
-		sitePlans: React.PropTypes.object.isRequired,
-		intervalType: React.PropTypes.string
-	},
+	static propTypes = {
+		basePlansPath: PropTypes.string,
```

Comment:
To be honest, I'm torn about whether this is a good idea. It's much more invasive than working with `propTypes` as it actually affects user-facing behavior.

In this particular case, it seems to make sense to default to `''`, but looking 2 components deeper, `PlansFeaturesMain.defaultProps.basePlansPath = null`! So do I "lift" the default from farther down the tree?

I'd be interested in having a conversation about this.